Source/EasyNetQ/Consumer/ConsumerDispatcher.cs

```diff
@@ -31,7 +32,7 @@ public ConsumerDispatcher(ConnectionConfiguration configuration)
                         }
                         else
                         {
-                            await Task.Delay(50);
+                            manualResetEvent.WaitOne();
```

Comment:
nice... I think the MRE should be disposed somewhere though

Reply:
True - not sure where it should be disposed. Becuase i think the queue should run until they are empty.

Reply:
In principle, nothing bad will happen if you do not dispose MRE.